modules/node/cms_node_api.e

```diff
@@ -220,6 +221,12 @@ feature -- Access: Node
 			Result := node_storage.nodes
 		end
 
+	node_revisions (a_node: CMS_NODE): LIST [CMS_NODE]
```

Comment:
Description missing.  Possibly list or node revisions for a given node `a_node'.